### PR TITLE
Add eslint-plugin-grommet and fix lint errors

### PR DESCRIPTION
**Pull request by taysea**
This PR adds the recommended configuration of `eslint-plugin-grommet` to grommet site. Additionally, I resolved the errors that were flagged by the linter (primarily missing alt-text on Images).

### CODE REVIEW

**Review by ericsoderberghp — APPROVED** [review]
Nice to see it doing its thing!

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "parser": "babel-eslint",
-  "extends": ["airbnb", "prettier"],
+  "extends": ["airbnb", "prettier", "plugin:grommet/recommended"],
   "plugins": ["prettier"],
```
Comment (halocline) [inline .eslintrc:4]: So this accesses the `recommended` property of the module config from `lib/index.js`?

Is the reason it gets placed in `extends` vs. `plugins` is it is auto applying? Whereas in `plugins` you are opting in a-la-carte?

File: src/screens/Home/Message.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const Message = ({ imageSrc, name, summary }) => (
       align="center"
       justify="center"
     >
-      <Image width={96} fit="cover" src={imageSrc} a11yTitle={name} />
+      <Image width={96} fit="cover" src={imageSrc} a11yTitle={name} alt={alt} />
```
Comment (halocline) [inline src/screens/Home/Message.js:15]: 👏

File: src/screens/Home/Tools.js [hunk 3]
```diff
@@ -31,12 +31,13 @@ const Tile = ({ desc, label, link, src }) => {
         {desc}
       </Paragraph>
       <Box flex />
-      <Image width={imgWidth} src={src} a11yTitle={label} />
+      <Image width={imgWidth} src={src} a11yTitle={label} alt={alt} />
```
Comment (halocline) [inline src/screens/Home/Tools.js:34]: 👏

**Review by halocline — APPROVED** [review]
Nice.

File: .eslintrc (hunk 1 above)
Comment (halocline) [reply .eslintrc:4]: Should this be added to the [README.md](https://github.com/grommet/eslint-plugin-grommet/blob/master/README.md)?

Comment (taysea) [reply .eslintrc:4]: Good call, I'll add some guidance in the README explaining this and how to use.

**Pull request merged**